doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -455,22 +455,24 @@ arguments used for configuration.  Here is a table describing these emulated dev
    * - ``virtio-net``
      - Virtio network type device, parameter should be appended with the format:
        ``virtio-net,<device_type>=<name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>],[mac_seed=<seed_string>]``.
-       The ``mac`` address is optional, ``name`` is the name of the TAP (or MacVTap) device.
-       ``vhost`` specifies vhost backend, otherwise the VBSU backend is used.
-       ``mac_seed=<seed_string>``
-       Set a platform-unique string as a seed to generate the mac address.  Each
-       VM should have a different "seed_string".  The "seed_string" can be
-       generated by the following method where $(vm_name) contains the name of the
-       VM you are going to launch.
+       The only ``device_type`` parameter that is supported at the moment is
```

Comment:
We avoid using phrases such as "currently" or "at the moment" that imply changes could be coming.  Just state what's supported now.
```suggestion
       The only supported ``device_type`` parameter is
```